eofconvention.fsx

```diff
@@ -0,0 +1,23 @@
+open System.IO
+
+let EOFIsEOL(path: string) = 
+    try
+        use sr = new StreamReader (path)
+        let filetext = sr.ReadToEnd()
+        (filetext |> Seq.last = '\n')
+        
+    with ex ->
```

Comment:
@realmarv when will this happen?

Reply:
I guess when the file is empty

Reply:
what exception type in particular? just putting a generic exception like this is a bad practice, use the specific type of exception that you got when you tested this

Reply:
Done